.github/workflows/buildAndPush.yml

```diff
@@ -0,0 +1,41 @@
+name: Build and push to dockerhub
```

Comment:
Thanks for providing this. In order to test that it works for me, I copied some files together in #112. I think, this file is not required any more but if you think you can add something to the existing file, please feel free to do so.